ui_framework/doc_site/src/views/tabs/tabs_example.js

```diff
@@ -33,5 +42,47 @@ export default props => (
         js={js}
       />
     </GuideSection>
+
+    <GuideSection
```

Comment:
We'll need a new example file which demonstrates just using the KuiTabs React component. I'm thinking it will look something like this:

```jsx
constructor() {
  this.tabs = [
    'Cobalt',
    'Dextrose',
    'Helium-3',
    'Monosodium Glutamate',
  ];

  this.state = {
    selectedTabIndex: 0,
  };
}

onSelectedTabChanged(index) {
  this.setState({
    selectedTabIndex: index,
  });
}

render() {
  <KuiTabs
    selectedTabIndex={this.state.selectedTabIndex}
    onSelectedTabChanged={this.onSelectedTabChanged}
  >
    {this.tabs}
  </KuiTabs>
}
```